src/collections/Form/index.d.ts

```diff
@@ -173,7 +173,7 @@ interface FormGroupProps {
 
 export const FormGroup: React.ComponentClass<FormGroupProps>;
 
-interface FormInputProps extends InputProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
+interface FormInputProps extends InputBaseProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
```

Comment:
This is the error I got before:

```
> tsc


176 interface FormInputProps extends InputProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
              ~~~~~~~~~~~~~~

node_modules/semantic-ui-react/dist/commonjs/collections/Form/index.d.ts(176,11): error TS2320: Interface 'FormInputProps' cannot simultaneously extend types 'InputProps' and 'ReactFormEvents<HTMLInputElement>'.
  Named property 'onChange' of types 'InputProps' and 'ReactFormEvents<HTMLInputElement>' are not identical.
```

So `ReactFormEvents<HTMLInputElement>` already includes the `onChange` prop with an incompatible signature. This is probably not the cleanest possible fix, but it works at least.